src/main/java/in/projecteka/consentmanager/common/Serializer.java

```diff
@@ -2,24 +2,36 @@
 
 import com.fasterxml.jackson.databind.DeserializationFeature;
```

Comment:
can you check swagger contract as well? make sure all dates are in the right format.

Reply:
When you grant a consent can you check whether HIP got notified successfullly and Consent Artefact stored successfully?